contracts/bonding/libraries/EarningsPool.sol

```diff
@@ -55,6 +55,16 @@ library EarningsPool {
         return earningsPool.claimableStake > 0;
     }
 
+    function activateStake(EarningsPool.Data storage earningsPool, uint256 _stake) internal {
+        earningsPool.totalStake = _stake;
+        earningsPool.claimableStake = _stake;
+    }
+
+    function deactivateStake(EarningsPool.Data storage earningsPool) internal {
```

Comment:
Let's add a function comment here.

Reply:
This function is going away in the next PR since we are no longer keeping track of active state on the earningspool.

instead we'll be using `setStake` and `setCommissions`